compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/Compiler.java

```diff
@@ -158,6 +158,7 @@ public void list(String sourcePackage) {
                 // skip the packages that were not loaded properly
                 packages.add(bLangPackage);
             }
+            dlog.resetErrorCount();
```

Comment:
IMO this should be added after this.compilerDriver.compilePackage(pkgNode) where we need to reset after compile code.

Reply:
Changed